src/bitfield.cpp

```diff
@@ -75,6 +79,30 @@ namespace libtorrent
 		}
 #endif // TORRENT_HAS_SSE
 
+#if TORRENT_HAS_ARM_NEON
+		if (aux::arm_neon_support)
+		{
+			for (int i = 0; i < words; ++i)
+			{
+				uint8x8_t in_val;
```

Comment:
I would prefer declaring the variables on the same line they're initialized. is there a reason not to do that? also, anything that can be "const" should be